rotkehlchen/chain/evm/accounting/aggregator.py

```diff
@@ -155,6 +164,40 @@ def get_accounting_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSett
             take=1,
             method='acquisition',
         )
+        swap_key = str(HistoryEventType.TRADE) + '__' + str(HistoryEventSubType.SPEND)
+        result[swap_key] = TxEventSettings(
+            taxable=True,
+            count_entire_amount_spend=False,
+            count_cost_basis_pnl=True,
+            method='spend',
+            take=2,
+            multitake_treatment=TxMultitakeTreatment.SWAP,
+        )
+        airdrop_key = str(HistoryEventType.RECEIVE) + '__' + str(HistoryEventSubType.AIRDROP)
+        airdrops_taxable = LedgerActionType.AIRDROP in pot.settings.taxable_ledger_actions
+        result[airdrop_key] = TxEventSettings(
+            taxable=airdrops_taxable,
+            count_entire_amount_spend=False,
+            count_cost_basis_pnl=False,
+            method='acquisition',
+            take=1,
+        )
+        reward_key = str(HistoryEventType.RECEIVE) + '__' + str(HistoryEventSubType.REWARD)
+        result[reward_key] = TxEventSettings(
+            taxable=True,
+            count_entire_amount_spend=False,
+            count_cost_basis_pnl=False,
+            method='acquisition',
+            take=1,
+        )
+        renew_key = str(HistoryEventType.RENEW) + '__' + str(HistoryEventSubType.NONE)
```

Comment:
why make this a generic default? Why not just add it to the only place this exists, ens module as an accountant?